### PR TITLE
DOC: Note MSVC compiler versions needed on Windows for runtests.py

### PR DESCRIPTION
**Pull request by mrKappen**

### CODE REVIEW

**Comment by charris** [comment]
I don't think this is true, we run tests on appveyor with different compiler versions.  On Windows you *do* need a compiler compatible with the Python version, for Python 2.7 that is the unmaintained, special download, MSVC 2008, for 3.5 (3.6?) I think it is 2015, and for 3.7 2017. Not sure about the precise versions however. @rgommers Do you know the correct versions?

**Comment by eric-wieser** [comment]
"distutils must be able to find a C compiler" would be a more accurate summary

File: runtests.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 """
 runtests.py [OPTIONS] [-- ARGS]
-
+Note: MSVC compiler versions needed on Windows for runtests.py to execute successfully
```
Comment (charris) [inline runtests.py]: How about just
```
Note: A C compiler is needed when running tests that require a build.
```
Note that it is possible the test the installed NumPy with the `--no-build` option.

**Review by mrKappen — COMMENTED** [review]
simplified comment

File: runtests.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 """
 runtests.py [OPTIONS] [-- ARGS]
-
+Note: A C compiler needed on Windows for runtests.py to execute successfully
```
Comment (charris) [inline runtests.py]: Not quite. Not all tests require a build (--no-build option),
```
Note: A C compiler needed for runtests.py to build and execute the numpy source.
```

**Comment by mattip** [comment]
I am -1 for this change. If any requirements to `runtest.py` are not found (pytest, a compiler, cython ...), an appropriate error should be raised rather than burdening users with extra text.

**Comment by rgommers** [comment]
This PR is now a single line which is not 100% correct. So let's close this. It may have helped you @mrKappen when you first ran into the problem, but in general `runtests.py` is not the right place to add build details. Thanks for contributing though!

**Pull request closed without merging**